samples/LibVLCSharp.Avalonia.Sample/App.xaml

```diff
@@ -7,7 +7,6 @@
     </Application.DataTemplates>
 
     <Application.Styles>
-        <StyleInclude Source="resm:Avalonia.Themes.Default.DefaultTheme.xaml?assembly=Avalonia.Themes.Default"/>
-        <StyleInclude Source="resm:Avalonia.Themes.Default.Accents.BaseLight.xaml?assembly=Avalonia.Themes.Default"/>
+		<FluentTheme />
```

Comment:
think there is a small formatting issue here caught by the github UI. tabs vs spaces?